code/datums/station_traits/neutral_traits.dm

```diff
@@ -97,7 +97,7 @@
 /datum/station_trait/glitched_pdas
 	name = "PDA glitch"
 	trait_type = STATION_TRAIT_NEUTRAL
-	weight = 15
+	weight = 8
```

Comment:
All this does is change the sound for when you get a PDA message. Is it that annoying?

Reply:
I've heard a lot of complaints, but a weight of 15 is VERY annoying, but more importantly it ruins the "fun" of rolling a random quirk.